### PR TITLE
question type classifier

### PR DESCRIPTION
**Pull request by LeonardPuettmannKern**
refinery
- [x] Tested by creator on refinery
- [x] Tested by reviewer on refinery
- [x] Ensured that output of brick conforms with refinery structure (to be checked by reviewer)

API
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by reviewer on localhost:8000/docs

common code
- [ ] Common code tested in notebook/ script by creator
- [x] Common code tested in notebook/ script by reviewer
- [x] Common code contains docstrings and type hints

additional points:
- [x] Docstring and README is existing
- [x] Import statements (in `__init__.py`)
- [x] (If necessary) Added dependency to requirements.txt
- [x] (If necessary) Added dependency to issue for refinery env [here](https://github.com/code-kern-ai/refinery/issues/166)
- [x] Published brick to Strapi CMS (locally)

Testing procedure: 
When testing in refinery, please ensure that the output of the brick conforms with the structure of refinery. 
For `extraction` bricks, this would be a tuple like `("label", span_start, span_end)`.
For `classification` bricks, this would be a string representing a label.
For `generator` bricks, this would either be a float, interger, string, boolean or a list, depending on the situation.

When testing the bricks, try to avoid using only one source of data. Meaning that you should not only use the `clickbait` sample
project, but also different texts with longer or more complex strings. 

A small refinery example project with a variation of texts called `bricks-test-data-project.zip` can be found in the bricks repository.

### CODE REVIEW

File: classifiers/__init__.py [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 from .lookup_lists import lookup_list
 
+from .query_check import (
```
Comment (FelixKirschKern) [inline classifiers/__init__.py:12]: `question_type` instead of `query_check` ?

File: classifiers/query_check/question_type_classifier/config.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        input_example=INPUT_EXAMPLE,
+        issue_id=344,
+        tabler_icon="ZoomQuestion",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "query_check"
+        ],  # first entry should be parent directory
```
Comment (FelixKirschKern) [inline classifiers/query_check/question_type_classifier/config.py:18]: also `question_type`?

File: classifiers/query_check/question_type_classifier/config.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                "ATTRIBUTE": {
+                    "selectionType": SelectionType.CHOICE.value,
+                    "addInfo": [
+                        BricksVariableType.ATTRIBUTE.value,
+                        BricksVariableType.GENERIC_STRING.value
+                    ]
+                },
+                "MODEL_NAME": {
+                    "selectionType": SelectionType.STRING.value,
+                    "defaultValue": "KernAI/multilingual-e5-communication-style",
```
Comment (FelixKirschKern) [inline classifiers/query_check/question_type_classifier/config.py:34]: defaultValue must be `"KernAI/multilingual-e5-question-type"`

File: classifiers/query_check/question_type_classifier/config.py (hunk 2 above)
Comment (LeonardPuettmannKern) [reply classifiers/query_check/question_type_classifier/config.py]: Yes, added it!

File: classifiers/query_check/question_type_classifier/config.py (hunk 3 above)
Comment (LeonardPuettmannKern) [reply classifiers/query_check/question_type_classifier/config.py]: Fixed. :+

File: classifiers/__init__.py (hunk 1 above)
Comment (LeonardPuettmannKern) [reply classifiers/__init__.py]: Jup, changed it

**Review by FelixKirschKern — APPROVED** [review]

**Pull request merged**